eras/alonzo/impl/cddl-files/alonzo.cddl

```diff
@@ -111,297 +341,188 @@ script_data_hash = $hash32 ; New
-; The values in the serialization are assumed to be ordered
-; lexicographically by their correpsonding key value.
-; See Plutus' `ParamName` for parameter ordering
-cost_model = [ 166*166 int64 ] ; New
-
-transaction_metadatum =
-    { * transaction_metadatum => transaction_metadatum }
-  / [ * transaction_metadatum ]
-  / int
-  / bytes .size (0..64)
-  / text .size (0..64)
+transaction_metadatum = {* transaction_metadatum => transaction_metadatum}
+                         / [* transaction_metadatum]
+                         / int
+                         / bytes .size (0 .. 64)
+                         / text .size (0 .. 64)
 
 transaction_metadatum_label = uint
-metadata = { * transaction_metadatum_label => transaction_metadatum }
-
-auxiliary_data =
-  metadata ; Shelley
-  / [ transaction_metadata: metadata ; Shelley-ma
-    , auxiliary_scripts: [ * native_script ]
-    ]
-  / #6.259({ ? 0 => metadata         ; Alonzo and future ; New
-      , ? 1 => [ * native_script ]
-      , ? 2 => [ * plutus_script ]
-      })
-
-
-
-vkeywitness = [ $vkey, $signature ]
-
-bootstrap_witness =
-  [ public_key : $vkey
-  , signature  : $signature
-  , chain_code : bytes .size 32
-  , attributes : bytes
-  ]
-
-native_script =
-  [ script_pubkey
-  // script_all
-  // script_any
-  // script_n_of_k
-  // invalid_before
-     ; Timelock validity intervals are half-open intervals [a, b).
-     ; This field specifies the left (included) endpoint a.
-  // invalid_hereafter
-     ; Timelock validity intervals are half-open intervals [a, b).
-     ; This field specifies the right (excluded) endpoint b.
-  ]
 
-script_pubkey = (0, addr_keyhash)
-script_all = (1, [ * native_script ])
-script_any = (2, [ * native_script ])
-script_n_of_k = (3, n: uint, [ * native_script ])
+; NEW:
+;   datum_hash: $hash32
+; 
+transaction_output = [address, amount : value, ? datum_hash : $hash32]
+
+; 
+; NEW: 
+;   3: [* plutus_script ]
+;   4: [* plutus_data ]
+;   5: redeemers
+; 
+transaction_witness_set = {? 0 : [* vkeywitness]
+                          , ? 1 : [* native_script]
+                          , ? 2 : [* bootstrap_witness]
+                          , ? 3 : [* plutus_script]
+                          , ? 4 : [* plutus_data]
+                          , ? 5 : redeemers}
+
+unit_interval = #6.30([1, 2])
+
+update = [proposed_protocol_parameter_updates, epoch]
+
+url = text .size (0 .. 64)
+
+value = coin / [coin, multiasset<uint>]
+
+vkeywitness = [$vkey, $signature]
+
+vrf_keyhash = $hash32
+
+withdrawals = {* reward_account => coin}
+
+genesis_key_delegation = (5, genesis_hash, genesis_delegate_hash, vrf_keyhash)
+
 invalid_before = (4, uint)
+
 invalid_hereafter = (5, uint)
 
-coin = uint
+move_instantaneous_rewards_cert = (6, move_instantaneous_reward)
 
-sub_coin = positive_interval
+; dns_name: An SRV DNS record
+; 
+multi_host_name = (2, dns_name)
+
+operational_cert = ($kes_vkey, uint, uint, $signature)
```

Comment:
This seems to have lost the field names
Previously it looked like this:
```cddl
operational_cert =
  ( hot_vkey        : $kes_vkey
  , sequence_number : uint
  , kes_period      : uint
  , sigma           : $signature
  )

```

Reply:
I think this is what the FIXME is about  here: https://github.com/IntersectMBO/cardano-ledger/pull/4778/files#diff-1e010fda09501224e5fc2da385ff69b74b167b3bef37013e4dd89655f0f83e25R67

We don't  seem to include the keys in the serialization, so I guess it's ok

Reply:
Yes, that's what that `FIXME` is about. I'm adding more of them in the PR for `babbage`, unfortunately, they are meant for changes to a dependent package. I will collect these and open issues on `cuddle` itself as well.